### DISCUSSION TITLE
Can't connect to 500

### DISCUSSION THREAD

**Discussion by deputylynch** [GerryDazoo/Slinger#298] [Q&A] [answered]
I tried to connect to my 500 and can't get anywhere!

**Comment by GerryDazoo** [comment, marked as answer]
Did you read the server logs before posting?
 It said you didn't have any valid IP address configured for you slingbox in the config.ini
Because you didn't do that it probed your network and found your slingbox at the IP address and port number you posted.
Now it's waiting for you to connect to the server with your media player.

So you haven't done anything wrong but I'd recommend fixing your config.ini wiith the right IP address and port number for your slingbox.

PS. Next time don't post screenshots of the server output. Cut and paste the text from the window.

**Reply by GerryDazoo** [reply]
Like I said, it's up and running and waiting for you to send a streaming request from your Media Player.
You don't have to plug in anything but it's recommended. Put your slingbox IP address and port number into your config.ini file.
And I suspect you also haven't put in place the Slingbox password which will be the next problem you will face when trying to stream.

I'd argue that you didn't follow that guide or you wouldn't be having this problem.

**Reply by deputylynch** [reply]
The only thing I saw in the guide that it said to change was  my password and to change the folder where slingbox is located of which I did both, but have no idea of where I would put my slingbox IP address and port number, nor what they are.

**Reply by GerryDazoo** [reply]
You posted a picture of that info and it is also printed out again in the server logs.

**Reply by deputylynch** [reply]
So 192.168.2.4 would go after ipaddress= ?
What about the port number?
